tests/BaseTest.php

```diff
@@ -55,4 +55,13 @@ public function getValidKeyName($key)
     {
         return preg_replace('|[^a-zA-Z0-9_\.! ]|', '', $key);
     }
+
+    protected function getCacheItemClass()
+    {
+        if (\PHP_VERSION_ID >= 80000) {
+            return 'Google\Auth\Cache\TypedItem';
```

Comment:
You can use this syntax (even if the class does not exists)
```suggestion
            return \Google\Auth\Cache\TypedItem::class;
```

(You can do it with a class import too).

Reply:
I can't because this library currently supports PHP 5.4, which does not support this syntax.

Reply:
Ohhh my bad, I didn't paid attention to such level backward compatibility.

My apologies.